src/test/java/RenameAPlaylist.java

```diff
@@ -1,53 +1,66 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.Keys;
 
 public class RenameAPlaylist extends BaseTest {
```

Comment:
**Naming Conventions in Java:**  Class names should be nouns

For example 

`NewNamePlayListTest`

![image](https://user-images.githubusercontent.com/104761482/200171831-75634559-69b8-430b-884e-e1f7429b40d0.png)